client/src/pages/Landing.tsx

```diff
@@ -1,27 +1,195 @@
+    //       <div className="flex flex-row md:my-36 md:mx-48 mx-9 h-2/3">
+    //         <div className="basis-1/2 ">
+    //           <h1
+    //             className="text-6xl font-[800] landing-title leading- tracking-tight"
+    //           >
+    //             A better way to <span className="text-primary-50"><br/>organize deadlines </span>
+    //           </h1>
+    //           <p className="text-xl font-poppins non-italic md:mt-10">
+    //               Lifeline takes the stress out of managing important dates for your courses. Start saving more time and always keep track of your deadlines in just a few clicks.
+    //             <span className="block md:mt-4">Start for free today to make your scheduling a breeze.</span>
+    //           </p>
+    //           <Button
+    //             onClick={routeChange}
+    //             variant="filled"
+    //           > 
+    //               Get Started 
+    //           </Button>
+    //         </div>
+    //         <div className="basis-1/2 md:mb-10">
+    //           <img className="w-3/4" src="illustration1.svg" alt="illustration 1" />
+    //         </div>
+    //       </div>
+    //     </div>
+    //   </div>
+    // </>
     <>
-      <main>
-        <h2 className="font-bold mt-10 text-emerald-500">
-          This is the landing page
-        </h2>
-        <span className="material-symbols-outlined">expand_more</span>
-      </main>
-
-      <Button variant="filled">Filled</Button>
-      <Button variant="filled" disabled><p>Filled</p> + <p>Disabled</p></Button>
-      <Button variant="tonal">Tonal</Button>
-      <Button variant="tonal" disabled>Tonal + Disabled</Button>
-      <Button variant="text" onClick={() => alert(111)}>Text</Button>
-      <Button variant="text" disabled>Text + Disabled</Button>
-
-      <nav>
-        <Link to="/upload">Get Started</Link>
-        <Link to="/workflow">Learn more</Link>
-      </nav>
+      <LogoBar/>
+      <div className="flex flex-row px-28 mt-28">
+        <div className="w-7/12 m-0 pl-12">
+          <div className="text-5xl font-poppins font-[800] leading- tracking-tight">
+            <h1>A better way to</h1>
+            <h1 className=" text-primary-50">organize deadlines</h1>
+          </div>
+          <div className="leading-relaxed font-poppins mt-10 w-11/12">
+            <p className="text-xl font-normal non-italic">
+              Lifeline takes the stress out of managing important dates for your
+              courses. Start saving more time and always keep track of your
+              deadlines in just a few clicks.
+              <span className="block">
+                Start for free today to make your scheduling a breeze.
+              </span>
+            </p>
+            <nav>
+              <div className="mt-[60px]">
+                <Link
+                  to="/upload"
+                >
+                  <Button
+                    onClick={routeChange}
```

Comment:
`<Link` and `<Button onClick` at the same time...? Probably keep only `<Link` is sufficient